docs/release-notes/self-hosted/self-hosted-v5.0.0.md

```diff
@@ -0,0 +1,93 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v5.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/3.9.16
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/4.0.28
+---
+
+# Self-hosted v5.0.0
+
+These release notes are for [Codacy Self-hosted v5.0.0](https://github.com/codacy/chart/releases/tag/5.0.0){: target="_blank"}, released on December 14, 2021. <!-- TODO Update release date -->
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Breaking changes
+
+This version of Codacy Self-hosted no longer includes support for legacy manual organizations.
+
+## Product enhancements
+
+-   Now, Codacy supports linting OpenAPI and AsyncAPI descriptions in either YAML or JSON files using [Spectral](https://stoplight.io/open-source/spectral/){: target="_blank"}. (CY-5088)
+
+-   You can now [use an organization coding standard](https://docs.codacy.com/v5.0/organizations/using-a-coding-standard/) to apply the same coding best practices, conventions, or security rules to a group of repositories. (CY-4654)
+
+    ![Organization coding standard](../images/cy-4654.png)
+
+## Bug fixes
+
+-   Fixed an issue that prevented non-logged users to see the file list of open-source repositories. (CY-5298)
```

Comment:
is this fix valid for SH? I don't know how public repos work on SH instance, but is it possible to visit a repo without logging in?

Reply:
Good question, I also think it isn't possible to view any Codacy Self-hosted page without being logged in. :thinking: Should we mark this issue as cloud-only @machadoit?

Reply:
Since machado is on vacation @pedrocodacy @lolgab ?

Reply:
Given that this is related to the files page migration, I would say that this issue never existed in SH installations anyways, therefore I would not publish the release note to avoid confusion.

In the meanwhile I tested opening a public repo page in sh and indeed it is not possible (you get a 404)

Reply:
I removed the release note for this issue. :heavy_check_mark: